### PR TITLE
Modified Scholly UI 

### PR DESCRIPTION
**Pull request by jfan0010**
# Summary
Modified Scholly UI to look different from Researchify, and user-friendlier to mobile viewers. 

# Type of change 
- [ ] Bug fix
- [ ] New feature
- [ ] Refactoring
- [ ] Documentation 
- [x] Idk, front-end? 

# Problem
The current deployed client-site looks identical to Researchify. Components overlapping when changing screen size (mobile view). 

# Solution description
Modified to display a simple base version of Scholly for user testing before the actual design of different themes implemented by team spring.  For deployed website, view: [DeployedWebsiteDemo](https://jfan0010.github.io/#/)
*Note that the landing page is currently dummy, will change in coming PR.

# Sreenshot
- landing page
image
- team member page
image
- publication page
image

# Risk
Low.

### CODE REVIEW

**Comment by vgoh1** [comment]
Great work - it's exciting to see the deployed site coming together! My thoughts:
Publications page:
- I feel like the subheadings are a bit small in the detailed view (authors, description etc) and could be bigger.
- Is it possible to not let the description span the entire length of the screen? It feels a bit weird on wider monitors, and I wonder if it looks better with more padding on the sides? And in the following case where it's just one sentence, it looks out of place haha
image
- Not sure how I feel about 'source' for the conference/journal title and you might need extra logic to say that it's conference/journal title 

Team members page:
- I think the members could be spaced out a bit more than they currently are?

**Comment by jfan0010** [comment]
And with Twitter account linked, the publication page will look like this:
image

* check out [another deployed site](https://jfan12094.github.io/#/publication) to view deployed site with twitter linked.

**Comment by jfan0010** [comment]
Styling modified. The subheadings (ie. authors, souces) are larger, the category type (above title) is darker, and the spacing of one-line description.

image

File: client/src/actions/team.js [hunk 1]
```diff
@@ -193,6 +193,8 @@ export const getGHAccessToken = (teamId, code) => async (dispatch) => {
   try {
     console.log(teamId);
     const response = await api.getGHAccessToken(teamId, code);
+    console.log(response)
```
Comment (aksh0001) [inline client/src/actions/team.js:196]: nit: remove.

File: client/src/components/dashboard/Dashboard.js [hunk 2]
```diff
@@ -21,10 +21,13 @@ import Webpages from './webpage/Webpages';
 import DeployPage from './deploy/DeployPage';
 
 const Dashboard = () => {
+  console.log('dashborad');
```
Comment (aksh0001) [inline client/src/components/dashboard/Dashboard.js:24]: same: prefer not to use logs like this in main, only for development

File: scholly/service/controllers/deploy.js [hunk 3]
```diff
@@ -17,9 +17,15 @@ const winston = require('winston');
  */
 async function handleDeployEvent(req, res) {
   const { teamId } = req.params;
+
+  console.log("!!!!!!!!!!!!!!", teamId)
```
Comment (aksh0001) [inline scholly/service/controllers/deploy.js:21]: ditto

File: scholly/service/controllers/deploy.js [hunk 4]
```diff
@@ -17,9 +17,15 @@ const winston = require('winston');
...
+
   winston.info(`Attempting to build and deploy application for team ${teamId}`);
   const data = req.body;
   const repoName = data.ghUsername + '.github.io';
+
+
```
Comment (aksh0001) [inline scholly/service/controllers/deploy.js:27]: nit: remove extra whitespaces

**Review by aksh0001 — APPROVED** [review]
LGTM on the readability. Will defer to @vgoh1 /others to address the aesthetics of the UI.

File: scholly/service/controllers/deploy.js (hunk 4 above)
Comment (jfan0010) [reply scholly/service/controllers/deploy.js]: removed

File: scholly/service/controllers/deploy.js (hunk 3 above)
Comment (jfan0010) [reply scholly/service/controllers/deploy.js]: removed

File: client/src/components/dashboard/Dashboard.js (hunk 2 above)
Comment (jfan0010) [reply client/src/components/dashboard/Dashboard.js]: removed

File: client/src/actions/team.js (hunk 1 above)
Comment (jfan0010) [reply client/src/actions/team.js]: removed.

**Review by AkashSaggar — APPROVED** [review]
UI looks good as a base to build on. Will really help team spring with their colours & layouts.

**Review by vgoh1 — APPROVED** [review]

**Pull request merged**